### ISSUE TITLE
Can't replicate the test results 

### ISSUE THREAD

**Issue by shukkkur** [apoorva-dave/LicensePlateDetector#29]
When when I run the code from DetectPlate.py on the first picture from README.md, this is the result I get
![image](https://user-images.githubusercontent.com/78250180/149452443-c4b284d8-b0f2-4fe4-ab31-707454dcf0d0.png)

On car6.jpg and car.png it works perfectly. 
Please help.

**Comment by matthiasgruber** [comment]
Yes. CV depends on the input shape; every picture should be scaled to the same dimension!

**Comment by jarulraj** [comment]
Thanks for sharing this helpful application, @apoorva-dave! Here is a notebook that also supports license plate detection:

https://github.com/georgia-tech-db/eva
https://github.com/georgia-tech-db/license-plate-recognition/blob/main/README.ipynb

![README_15_0](https://user-images.githubusercontent.com/5521975/212554733-09d6ab55-fff8-4b65-b142-906f087ff917.png)
